src/qt/forms/editaddressdialog.ui

```diff
@@ -53,6 +53,16 @@
        </property>
       </widget>
      </item>
+     <item row="2" column="1">
+      <widget class="QLabel" name="errorMessage">
+       <property name="styleSheet">
+        <string notr="true">color:red;</string>
```

Comment:
Is hardcoded red color visually compatible with dark theme? Maybe add some relevant screenshots?

Reply:
There is a screenshot posted by @jonatack here:
https://github.com/bitcoin-core/gui/pull/534#pullrequestreview-860753411

But this style is already used in `sendcoinsdialog.ui` and `sendcoinsdialog.cpp` for invalid custom change addresses and insufficient funds , so it seems to be a well tested configuration.